src/callbacks/onLayout/initStudyDayControl/startAnimation.js

```diff
@@ -48,7 +48,8 @@ export default function startAnimation() {
             var vals = raw[m + '_raw'];
 
             // Did currentDay occur while participant was enrolled?
-            if (vals.length) {
+            if (vals && vals.length) {
+                // && [config.x.column, config.y.column].includes(m)) { // out-of-range should be calculated study day of with x- and y-axis measures
```

Comment:
Ugh. This is all pretty gross, would probably be good to modularize enough to re-use the code in the original calculations ...